_blogposts/2020-11-12-rescript-the-road-ahead.mdx

```diff
@@ -0,0 +1,80 @@
+---
+author: maxim
+date: "2020-11-12"
+previewImg:
+category: syntax
+badge: roadmap
+title: "Editor Support, Custom Operators and More"
+description: |
+  Update on what we're doing around the end of 2020 and early next year.
+canonical: https://rescript-lang.org/blog/rescript-the-road-ahead
+---
+
+import Video from "components/Video"
+
+## Upcoming Improvements
+
+As we're approaching the end of 2020, we'd like to inform our community of our remaining tasks for the year and early next year.
+
+## Editor Integration
+
+In the coming days, Cristiano and Cheng Lou will finally be ready to unveil the new VSCode ReScript experience, including diagnosis, autocompletion, jump to definition and type hints!
+
+<Video withShadow={true} src="https://player.vimeo.com/video/477758754" />
+
+Patrick and Maxim will also help polishing the same feature set for Vim and Sublime Text.
+
+## Compiler & Build
+
+Hongbo continues to fine tune the compiler and build system for performance and stability. There are several improvements coming down the line. Expect your build to keep getting faster and more robust!
+
```

Comment:
The current priority is focused on the user experience of pinned packages support